src/App.js

```diff
@@ -1,80 +1,16 @@
 import React from 'react';
 import './App.css';
-// Move each BEM block to a separate component (file) and import them here
+
+import Welcome from './components/Welcome/Welcome';
+import Header from './components/Header/Header';
+import Article from './components/Article/article';
```

Comment:
Follow files naming style that has on your project
```suggestion
import Article from './components/Article/Article';
```